src/xmipp/libraries/data/symmetries.cpp

```diff
@@ -2223,37 +2225,36 @@ double SymList::computeDistance(double rot1, double tilt1, double psi1,
 }
 
 void SymList::breakSymmetry(double rot1, double tilt1, double psi1,
-                              double &rot2, double &tilt2, double &psi2
-                              )
+                            double &rot2, double &tilt2, double &psi2)
 {
     Matrix2D<double> E1;
     Euler_angles2matrix(rot1, tilt1, psi1, E1, true);
     static bool doRandomize=true;
     Matrix2D<double>  L(3, 3), R(3, 3);  // A matrix from the list
 
-    int i;
+    static unsigned seed = 0;
     if (doRandomize)
     {
-        srand ( time(nullptr) );
+    	seed = std::chrono::system_clock::now().time_since_epoch().count();
         doRandomize=false;
     }
+
     int symOrder = symsNo()+1;
-    //std::cerr << "DEBUG_ROB: symOrder: " << symOrder << std::endl;
-    i = rand() % symOrder;//59+1
-    //std::cerr << "DEBUG_ROB: i: " << i << std::endl;
+    std::default_random_engine generator (seed);
```

Comment:
I dare say that this change will break our tests

Reply:
Restored